hedera-mirror-rest/cache.js

```
@@ -78,10 +78,13 @@ export class Cache {
       return loader(keys);
     }
 
-    const buffers =
+    const mappedKeys = _.map(keys, keyMapper);
+    const buffers = mappedKeys.length === 0 ? [] :
```

Would prefer that we revert this and add a fail fast at the top before !ready:

```javascript
    if (_.isEmpty(keys)) {
        return [];
    }
```

---

Thought maybe `keyMapper` in `_.map()` could delete entries, potentially resulting in an empty array of keys. More digging does not show that being the case. So, reverted and added the suggested fail fast statement.